hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieMultiTableDeltaStreamer.java

```diff
@@ -296,6 +297,11 @@ public static void main(String[] args) throws IOException {
         + "outstanding compactions is less than this number")
     public Integer maxPendingCompactions = 5;
 
+    @Parameter(names = {"--max-pending-clustering"},
```

Comment:
out of curiosity. how come we have maxPendingCompaction/Clustering property defined as first class config for multiTableDeltaStreamer, but don't see the properties to enable/disable them. I assume those are fetched from property file for each source. So, why not fetch these properties also from the property file? I know this is not specific to clustering, but it was how compaction was defined.

Reply:
That's a valid point. I followed how we defined for compaction. We could have enable/disable configs as first class configs. Anything related to a particular feature/service could be passed in property file or as `--hoodie-conf`. This will avoid config bloating in delta streamer. 

@bvaradar Was there any reason for doing it this way?

Reply:
@pratyakshsharma ^